packages/venia-ui/lib/util/formValidators.js

```diff
@@ -106,3 +106,13 @@ export const validateConfirmPassword = (
 ) => {
     return value === values[passwordKey] ? SUCCESS : 'Passwords must match.';
 };
+
+export const validateDifferentCurrentPassword = (
```

Comment:
This is a bit too single use, but I can see it's just the copied inverse of `validateConfirmPassword`. I probably would have gone more `isEqualToField` and `isNotEqualToField` (informed doesn't appear to have not operators for validation).

Reply:
What should I change to good?

Reply:
Updated.